src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
```

Comment:
```suggestion
  font-family: Arial, sans-serif;
```

move to html selector

Reply:
what is it?

Reply:
html {
  font-family: Arial, sans-serif;
}